src/observablePlaylist.ts

```diff
@@ -81,30 +83,50 @@ export namespace ObservablePlaylist {
 
     env.printQueueRequest.subscribe(() => {
       if (queue.length === 0 || queue.length <= currentQueueIndex) {
-        env.sendMessage.next('The queue is empty...')
+        env.sendMessage.next('The queue is empty 👀')
       } else {
-        const printedQueue: string[] = []
         const start = currentQueueIndex
-        const end = currentQueueIndex + 5
-        for (let i = start; i <= end; i++) {
-          const curr = queue[i]
-          if (curr) {
-            if (curr.removed) {
-              printedQueue[i] = `[${i}] ~~${curr.name}~~`
-            } else {
-              printedQueue[i] = `[${i}] ${curr.name}`
-            }
-          }
-        }
-
-        printedQueue[currentQueueIndex] = `${printedQueue[currentQueueIndex]} <-- Now playing`
-        if (end < queue.length) {
-          printedQueue.push('...')
-        }
-
-        env.sendMessage.next(printedQueue.join('\n\n'))
+        var message: MessageWithReactions = printQueue(start, queue, currentQueueIndex, currentPage)
+        env.sendMessage.next(message)
       }
     })
+
+    env.reprintQueueOnReaction.subscribe(prevMsg => {
```

Comment:
Kan awaita den här med en try-catch om du gillar await-syntaxen mer.
```suggestion
    env.reprintQueueOnReaction.subscribe(async prevMsg => {
      try {
        const collected = await prevMsg.awaitReactions({
          filter: (reaction, user) => {
            const emojiName = reaction.emoji.name
            return emojiName != null ? ['⏪', '◀', '▶', '⏩'].includes(emojiName) && !user.bot : false
          },
          max: 1,
          time: 60000,
          errors: ['time']
        })

        const reaction = collected.first()
        if (reaction?.emoji.name === '▶') {
          const start = currentQueueIndex + pageSize * ++currentPage
          const newMessage: MessageWithReactions = printQueue(start, queue, currentQueueIndex, currentPage)
          env.editMessage.next(new EditedMessage(prevMsg, newMessage))
        } else if (reaction?.emoji.name === '◀') {
          const start = currentQueueIndex + pageSize * --currentPage
          const newMessage: MessageWithReactions = printQueue(start, queue, currentQueueIndex, currentPage)
          env.editMessage.next(new EditedMessage(prevMsg, newMessage))
        } else if (reaction?.emoji.name === '⏪') {
          currentPage = currentPage - 2
          const start = currentQueueIndex + pageSize * currentPage
          const newMessage: MessageWithReactions = printQueue(start, queue, currentQueueIndex, currentPage)
          env.editMessage.next(new EditedMessage(prevMsg, newMessage))
        } else if (reaction?.emoji.name === '⏩') {
          currentPage = currentPage + 2
          const start = currentQueueIndex + pageSize * currentPage
          const newMessage: MessageWithReactions = printQueue(start, queue, currentQueueIndex, currentPage)
          env.editMessage.next(new EditedMessage(prevMsg, newMessage))
        }
      } catch(e) {
        logger.error(e)
        prevMsg.reactions.removeAll()
      }
    })
```

Kan också vara bra att lägga in lite loggning vid catch -> se ovan

Reply:
Fixade loggningen, men hade kvar then mönstret med promises